builder/powervs/step_create_network.go

```diff
@@ -54,3 +85,47 @@ func (s *StepCreateNetwork) Cleanup(state multistep.StateBag) {
 			"Error cleaning up network. Please delete the network manually: %s", *net.Name))
 	}
 }
+
+func (s *StepCreateNetwork) createDHCPNetwork(state multistep.StateBag) error {
+	dhcpClient := state.Get("dhcpClient").(*instance.IBMPIDhcpClient)
+
+	dhcpServerCreateParams := models.DHCPServerCreate{
+		Name: core.StringPtr("packerdhcpnetwork"),
+	}
+
+	dhcpServer, err := dhcpClient.Create(&dhcpServerCreateParams)
+	if err != nil {
+		return fmt.Errorf("error failed to create DHCP server: %v", err)
+	}
+
+	if dhcpServer.ID == nil {
+		return fmt.Errorf("error created DHCP server ID is nil")
+	}
+	state.Put("dhcpServerID", *dhcpServer.ID)
+
+	currentTime := time.Now()
+	var networkID string
+	for {
+		dhcpServerDetails, err := dhcpClient.Get(*dhcpServer.ID)
+		if err != nil {
+			return err
+		}
+		if dhcpServerDetails.Network != nil && dhcpServerDetails.Network.ID != nil {
+			networkID = *dhcpServerDetails.Network.ID
+			break
+		}
+		if time.Since(currentTime) > DHCPServerActiveTimeOut {
+			return fmt.Errorf("error DHCP server did not become active even after %f min", DHCPServerActiveTimeOut.Minutes())
+		}
+		time.Sleep(DHCPServerInterval)
```

Comment:
lets add some text in the for loop to show the progress

Reply:
Added.